platform/viewer/.env

```diff
@@ -6,5 +6,6 @@
 # Be careful not to commit anything sensitive to source control.
 #
 
-PUBLIC_URL=/
-APP_CONFIG=config/default.js
+REACT_APP_PUBLIC_URL=/
+REACT_APP_APP_CONFIG=config/default.js
+REACT_APP_USE_HASH_ROUTER=false
```

Comment:
We're not using CRA, so you shouldn't need to prefix these env vars with `REACT_APP_`